user/urls.py

```diff
@@ -1,10 +1,9 @@
 from django.urls import path
-from user.views import CreateUserView, CreateTokenView, ManageUserView
+from user.views import CreateUserView, ManageUserView
 
 app_name = "user"
 
```

Comment:
Add a JWT support for the project.

Reply:
JWT is already added, you can check for these lines inside my settings.py:

`
"DEFAULT_AUTHENTICATION_CLASSES": (
 	"rest_framework_simplejwt.authentication.JWTAuthentication",
)

SIMPLE_JWT = {
    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=120),
    "REFRESH_TOKEN_LIFETIME": timedelta(days=7),
    "ROTATE_REFRESH_TOKENS": True,
}
`

These lines inside urls.py:

` 
path(
        "api/token/", TokenObtainPairView.as_view(), name="token_obtain_pair"
    ),
    path(
        "api/token/refresh/", TokenRefreshView.as_view(), name="token_refresh"
    ),
`


Also, you can run my pull request locally and see that JWT is working fine.

Please check again and let me know if something is wrong.